substrate/frame/staking/src/benchmarking.rs

```diff
@@ -274,16 +274,15 @@ benchmarks! {
 
 		let stash = scenario.origin_stash1.clone();
 		let controller = scenario.origin_controller1.clone();
-		let amount = origin_weight - scenario.dest_weight;
```

Comment:
Would this impact the worst case weight calculation for updating bags list?

Reply:
wrt https://github.com/paritytech/polkadot-sdk/pull/3811/files?diff=split&w=0#r1717563991, an alternate way would be to keep this existing benchmark and add the chill weight to it for worst case weight of `unbond`.